src/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue

```diff
@@ -212,9 +237,21 @@ export default {
         },
       ],
       includeOpts: [
-        {id: 1, param: 'onlyCurrent', label: 'Current Employees Only'},
-        {id: 2, param: 'currentAndPast', label: 'Current and Past Employees'},
-        {id: 3, param: 'onlyPast', label: 'Past Employees Only'},
+        {
```

Comment:
I feel like this Include dropdown and its options are getting repeated. can't we refactor all to a include-dropdown component of its own ?